src/utilities/isPalindrome.js

```diff
@@ -0,0 +1,9 @@
+export function isPalindrome(text) {
+  const palindrome = text
+    .replace(/[^a-zA-Z]/g, '')
```

Comment:
Can you solve this without a regular expression?
They are usually very hard to read, and you should avoid them if you can.

![image](https://user-images.githubusercontent.com/15181215/222013206-438ddc70-0027-4e3e-b69e-c00ebdc133ea.png)